desktop/cmp/persistenceManager/Types.ts

```diff
@@ -0,0 +1,28 @@
+import {PlainObject} from '@xh/hoist/core';
+import {LocalDate} from '@xh/hoist/utils/datetime';
+
+export interface PersistenceView<T extends PlainObject = PlainObject> {
+    id: number;
+    acl: string;
+    archived: boolean;
+    dateCreated: LocalDate;
+    description: string;
+    lastUpdated: LocalDate;
+    lastUpdatedBy: string;
+    name: string;
+    owner: string;
+    token: string;
+    type: string;
+    group: string;
+    isShared: boolean;
+    value: T;
+}
+
+export interface TreeView {
```

Comment:
Let's think about the name for this interface. Maybe `PersistenceViewTree`? Or just `ViewTree` but I'm afraid that could get confused with Cube Views.